app/src/main/java/com/example/login/ui/ResultFragment.kt

```diff
@@ -0,0 +1,53 @@
+package com.example.login.ui
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.activityViewModels
+import androidx.navigation.Navigation
+import com.example.login.R
+import com.example.login.databinding.FragmentResultBinding
+import com.example.login.viewmodel.UserAuthorizationViewModel
+
+class ResultFragment : Fragment() {
+    private lateinit var binding: FragmentResultBinding
+    private val userAuthorizationViewModel: UserAuthorizationViewModel by activityViewModels()
+
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        binding = FragmentResultBinding.inflate(inflater, container, false)
+
+        initUserInfo()
+        binding.btnTryAgain.setOnClickListener {
+            view?.let { it1 -> Navigation.findNavController(it1).navigate(R.id.resultToIntroduce) }
+        }
+
+        return binding.root
+    }
+
+    private fun initUserInfo(){
+        userAuthorizationViewModel.userNameModel.observe(this, { name ->
```

Comment:
Read about binding and pass data with binding

Reply:
done